client/src/__tests__/search.test.js

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import {render, screen, waitFor} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import "mutationobserver-shim"
+import Search from '../components/Search'
+
+jest.setTimeout(30000);
+
+test('test if "loading" message appears after submitting a search', async () => {
+
+  render(<Search />)
+
+  screen.getByRole('textbox', { name: 'Name' });
+  userEvent.type(screen.getByRole('textbox', { name: 'Name' }), '1335 ASTOR');
+  userEvent.click(screen.getByRole('button', {name: /Submit/i}));
+
+  await waitFor(() => screen.findByText(/Loading/i), {timeout:5000});
```

Comment:
What is the timeout for here?  Shouldn't the loading message show up immediately upon form submission and then disappear when results are returned?

Reply:
I removed the timeout.